src/index.js

```diff
@@ -91,6 +95,25 @@ export default class AutoCheckElement extends HTMLElement {
   }
 }
 
+function setLoadingState(event: Event) {
+  const input = event.currentTarget
+  if (!(input instanceof HTMLInputElement)) return
+
+  let message = 'Verifying…'
+  const setValidity = text => (message = text)
+  input.dispatchEvent(
+    new CustomEvent('auto-check-input', {
```

Comment:
Perhaps call it start? this happens whenever value changes via with/without input.

```suggestion
    new CustomEvent('auto-check-start', {
```